vue/components/ui/organisms/header/announcement-modal.vue

```diff
@@ -0,0 +1,281 @@
+<template>
+    <transition name="fade">
+        <div
+            class="announcements-container"
+            v-bind:class="{ visible: visible }"
+            v-show="isVisible"
+            v-on:click.stop

```

Comment:
Why is this needed? Can't recall.

Reply:
Prevent the announcements of closing when clicking inside the announcements modal